templates/trt-llm/packages/client.py

```diff
@@ -122,34 +56,99 @@ def load_server_and_model(self, env: dict):
             time.sleep(2)
             continue
 
-    @staticmethod
-    def stream_predict(user_data: UserData):
-        """Static method to yield predictions or errors based on input and a streaming user_data queue."""
+    def start_server(
+        self,
+        mpi: int = 1,
+        env: dict = {},
+    ):
+        """Triton Inference Server has different startup commands depending on
+        whether it is running in a TP=1 or TP>1 configuration. This function
+        starts the server with the appropriate command."""
+        def _build_server_start_command(
+            mpi: int = 1,
+            env: dict = {},
+        ):
+            base_command = [
+                "tritonserver",
+                "--model-repository", str(self._model_repository_dir),
+                "--grpc-port", str(GRPC_SERVICE_PORT),
+                "--http-port", str(HTTP_SERVICE_PORT)
+            ]
+
+            if mpi == 1:
+                # return base_command + ["--log-verbose", "1"]
```

Comment:
Maybe explain how the commented code could be used for getting more logs from triton server.